test_libs/pyspec/eth2spec/test/block_processing/test_process_block_header.py

```diff
@@ -0,0 +1,85 @@
+from copy import deepcopy
```

Comment:
Had to manually check this file against the `dev` branch due to the file being moved. Don't think I had to port any tests. Just a few modifications to names/etc